src/table/table.jsx

```diff
@@ -27,6 +27,9 @@ const Table = React.createClass({
     selectable: React.PropTypes.bool,
     style: React.PropTypes.object,
     wrapperStyle: React.PropTypes.object,
+    headerTableInlineStyle: React.PropTypes.object,
```

Comment:
Could you remove the `Inline` part of the name to use the same convention as other component?